polaris/polaris/sep12/customer.py

```diff
@@ -45,7 +45,9 @@ def get(account: str, request: Request) -> Response:
             response_data = rci.get(
                 {
                     "id": request.GET.get("id"),
-                    "account": request.GET.get("account"),
+                    # anchors need to be able to filter out customers who weren't
+                    # created by the following SEP-10 JWT source account.
+                    "account": account,
```

Comment:
I think we need to pass two different accounts here, one would be the SEP-10 pubkey, and the other would be the `account` query parameter. They can be different, as mentioned [here](https://github.com/stellar/stellar-protocol/issues/676#issuecomment-656252392).

Reply:
They could be in the future but right now Polaris expects them to be the same.

Reply:
We could pass the JWT source account to the `CustomerIntegration.get()` function under the `authenticated_account` key, and leave `account` as `None` to differentiate between the request parameter and the JWT source account?

Reply:
I still think we need two, it might be confusing to pass a value for the query parameter `account` and receive a response based on a different account (from SEP-10). I suggest either returning an error if they are different, or passing it to the integration function so the developer can decide to raise an error if they're different.

Reply:
> We could pass the JWT source account to the CustomerIntegration.get() function under the authenticated_account key, and leave account as None to differentiate between the request parameter and the JWT source account?

Yes, sorry my previous reply was to your previous message.
I think it's better to pass both to the integration function.

Reply:
They will always match for now, but we can pass both so we can avoid the breaking change post-1.0